sdk/devcenter/Azure.Developer.DevCenter/src/autorest.md

```diff
@@ -0,0 +1,25 @@
+# Azure SDK Code Generation for Data Plane
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+openapi-type: data-plane
+azure-arm: false
+tag: v2022-03-01-preview
+eol: crlf
+license-header: MICROSOFT_MIT_NO_VERSION
+public-clients: true
+
+input-file:
+  - ~\source\repos\azure-devtest-center\src\sdk\specification\devcenter\data-plane\Microsoft.DevCenter\preview\2022-03-01-preview\devcenter.json
+  - ~\source\repos\azure-devtest-center\src\sdk\specification\devcenter\data-plane\Microsoft.DevCenter\preview\2022-03-01-preview\devbox.json
+  - ~\source\repos\azure-devtest-center\src\sdk\specification\devcenter\data-plane\Microsoft.DevCenter\preview\2022-03-01-preview\environments.json
```

Comment:
These links are not correct and that's the reason of [this](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1910682&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=a880e989-7d1a-5c96-a41f-d540b383cc43&l=59) failure.